libmscore/timesig.cpp

```diff
@@ -352,8 +355,9 @@ void TimeSig::layout()
             // centering of parenthesis so the middle of the parenthesis is at the divisor marking level
             int centerY = yoff/2 + _spatium;
             int widestPortion = numRect.width() > denRect.width() ? numRect.width() : denRect.width();
-            pointLargeLeftParen = QPointF(-_spatium, centerY);
-            pointLargeRightParen = QPointF(widestPortion + _spatium, centerY);
+            pointLargeLeftParen = QPointF(-0.7  * _spatium * _scale.width() , centerY);
```

Comment:
Just for clarity, where does this `-.07` come from?
dito for the `0.3` next line.

Reply:
Since the coords for these large ( ) glyphs are set manually (no metrics really involved) the -0.7 and  +0.3 are "magic" hard-coded tuning. Now that the code includes _scale.width() into the calculation, just a portion of _spatium is needed to adjust the positions, and it's different for the opening and closing parenthesis.
These numbers could be replaced by getting the () glyphs' metrics (bbox, _advance_ etc) and do proper calculations from there, but frankly I don't think it's worth coding. The utility of these parentheses is very limited and IMO all the alternative text for time signatures needs reworking. 
But since the ( ) are there, I did this small fix to at least make them work as expected.